upgrade/README.md

```diff
@@ -103,14 +103,20 @@ Upgrade sequence: (4.0.4.1 to 4.0.4.2)
 16. Restart all pods:
       - `kubectl -n opsmx-isd scale deploy -l app=oes --replicas=0` Wait for a min or two
       - `kubectl -n opsmx-isd scale deploy -l app=oes --replicas=1` Wait for all pods to come to ready state
-      
-17. If you enabled new Insights feature in step 8, please follow the post installation steps listed [here](https://docs.google.com/document/d/1FgbvGeylTmWKBFKZNs2mMkKlkxHpyzPMEy5wJCaKSxk/edit#heading=h.odfvfs38x0e3)
+        
+17. Need to add the below code manullay in the `default/profiles/echo-local.yml` for echo pods to come to ready state.
```

Comment:
Add the below configuration (if not already psesent)  in the `default/profiles/echo-local.yml` for echo pods